c++/src/kj/async-io.h

```diff
@@ -56,16 +56,16 @@ class AsyncInputStream: private AsyncObject {
   // Asynchronous equivalent of InputStream (from io.h).
 
 public:
-  virtual Promise<size_t> read(void* buffer, size_t minBytes, size_t maxBytes);
+  virtual Promise<size_t> read(ArrayPtr<byte> buffer, size_t minBytes);
```

Comment:
Maybe not related to this specific change but I've long thought this never should have been virtual, it should just be based on tryRead... just never got around to changing it.

Reply:
sure, can do. I'm touching this anyway.